tests/opentelemetry/test_trace.py

```diff
@@ -17,6 +17,7 @@ def test_otel_compatible_tracer_is_returned_by_tracer_provider():
     assert isinstance(otel_compatible_tracer, opentelemetry.trace.Tracer)
 
 
+@flaky(1735812000)
```

Comment:
Same here. This test doesn't do anything special. If it's flaky then it's likely an issue with how capture snapshot tests. We could use `@pytest.mark.snapshot(wait_for_num_traces=1)` to increase reliability

Reply:
https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.status%3Afail%20%40test.name%3Atest_otel_start_span_with_default_args&index=citest&start=1704658024208&end=1704744424208&paused=false